SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialSnapshotTest/SpatialSnapshotTestPart2SettingsOverride.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialSnapshotTestPart2SettingsOverride.h"
+#include "Editor/EditorPerformanceSettings.h"
+#include "Settings/LevelEditorPlaySettings.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "SpatialGDKSettings.h"
+
+/**
+ * This test checks that the test settings overridden in the .ini file have been set correctly
+ *
+ * Requires TestOverridesSpatialSnapshotTestPart2Map.ini in \Samples\UnrealGDKTestGyms\Game\Config directory with the following values:
+ *			[/Script/UnrealEd.LevelEditorPlaySettings]
+ *			PlayNumberOfClients=1
+ *			PlayNetMode=PIE_Client
+ *
+ *			[/Script/EngineSettings.GeneralProjectSettings]
+ *			bSpatialNetworking=False
+ */
+
+ASpatialSnapshotTestPart2SettingsOverride::ASpatialSnapshotTestPart2SettingsOverride()
+	: Super()
+{
+	Author = "Victoria Bloom";
+	Description = TEXT("Spatial Snapshot Test Part 2 - Settings Override");
+}
+
+void ASpatialSnapshotTestPart2SettingsOverride::PrepareTest()
```

Comment:
I assume we will want to add these tests to the corresponding generated maps?
It may be good to sync with Valentyn on his work to port older maps to the new generated format.

Reply:
Yes, I was planning to add them in v4.25 to the existing maps but if Valentyn is also working on porting the older maps I will check with him.